vm/test.c

```diff
@@ -34,10 +34,11 @@ static void register_functions(struct ubpf_vm *vm);
 
 static void usage(const char *name)
 {
-    fprintf(stderr, "usage: %s [-h] [-j|--jit] [-m|--mem PATH] BINARY\n", name);
+    fprintf(stderr, "usage: %s [-h] [-j|--jit] [-V|--verify] [-m|--mem PATH] BINARY\n", name);
```

Comment:
Because usually `-v` is used for verbosity control. But if that's not something that'll ever get added to this program, then I'll switch verification to `-v`.

Reply:
Maybe let's go for `-v` for verify and we'll take `-d` if we ever add a debug mode. I doubt we'll need several levels of debug before long. In any case, we can always change later on (until we make a release I guess :-) ).